website/templates/project/nodes_privacy.mako

```diff
@@ -0,0 +1,121 @@
+<div id="nodesPrivacy" class="modal fade">
+    <div class="modal-dialog modal-md">
+        <div class="modal-content">
+            <div class="modal-header">
+                <button type="button" class="close" data-dismiss="modal" data-bind="click: clear" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+                <h3 class="modal-title" data-bind="text:pageTitle"></h3>
+            </div>
+
+            <div class="modal-body">
+
+                <!-- warning page -->
+
+                <div data-bind="if: page() == 'warning'">
+                    <span data-bind="html:message"></span>
+                </div>
+
+                <!-- end warning page -->
+
+                <div data-bind='visible:page() === "select"'>
+                    <div class="row">
+                        <div class="col-md-10">
+                            <div class="m-b-md box p-sm">
+                                <span data-bind="html:message"></span>
+                            </div>
+                        </div>
+                    </div>
+                    <div>
+                        Select:&nbsp;
+                        <a data-bind="click:selectAll">Make all public</a>
+                        &nbsp;|&nbsp;
+                        <a data-bind="click:selectNone">Make all private</a>
+                    </div>
+                        <div class="tb-row-titles">
+                            <div style="width: 100%" data-tb-th-col="0" class="tb-th">
+                                <span class="m-r-sm"></span>
+                            </div>
+                        </div>
+                        <div class="osf-treebeard">
+                            <div id="grid">
+                                <div class="spinner-loading-wrapper">
+                                    <div class="logo-spin logo-md"></div>
+                                    <p class="m-t-sm fg-load-message"> Loading projects and components...  </p>
+                                </div>
+                            </div>
+                            <div class="help-block" style="padding-left: 15px">
+                                <p id="configureNotificationsMessage"></p>
+                            </div>
+                        </div>
+                </div>
+                <!-- end select projects page -->
+
+                <!-- addon and projects changed warning page -->
+
+                <div data-bind="if: page() == 'addon'">
+
+                    <div class="m-b-md box p-xs" data-bind="visible: changedAddons().length > 0">
+                        <span class="text-bigger">The following addons will be effected by this change:</span>
+                        <ul data-bind="foreach: { data: changedAddons, as: 'item' }">
```

Comment:
Can you use `self.message.addonWarning` instead of writing out the message in the template? (Same with the other two messages)